src/main/scala/diplomacy/Nodes.scala

```diff
@@ -578,7 +586,7 @@ class IdentityNode[D, U, EO, EI, B <: Data](imp: NodeImp[D, U, EO, EI, B])()(imp
   extends AdapterNode(imp)({ s => s }, { s => s })
 {
   override def description = "identity"
-  protected override val identity = true
+  override final def identity = true
```

Comment:
`final` is a bit dramatic, is it not?

Reply:
I see now that you were prompted to add these by a prior reviewer.

I think a lot hinges on what this 'identity: Boolean' means. This PR uses it to mean not only that the parameters are identity, but that the wires are an identity. With that definition, it definitely is not correct to say that all IdentityNodes are circuit identity.

Perhaps 'identity' should be named something that makes it clear the circuit is also doing nothing?

Reply:
Actually, I now recall that IdentityNodes differ from AdapterNodes in that it directly cross connects the bundles for you. Therefore, there can indeed be no hardware in the middle, so this is safe.